roles/logging/tasks/filebeat.yml

```diff
@@ -4,10 +4,14 @@
     pkg: "filebeat={{ logging.forward.version }}"
   notify: restart filebeat
   when: os == "ubuntu"
-  
+
 - name: install filebeat yum
   yum:
     name: "filebeat"
+  register: result
```

Comment:
for shits and giggles add the retry for ubuntu above as well.   I'm told ubuntu repos are also a trash fire.